documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -106,27 +106,27 @@ spec:
   map:
     networks:
       - source:
-          id: <network_ref> <3>
+          id: <source_network> <3>
         destination:
           type: pod
           name: pod
           namespace: <target_namespace> <4>
     datastores:
       - source: <5>
-          id: <datastore_ref> <6>
+          id: <source_data_store> <6>
         destination:
           storageClass: standard
   vms:
-    - id: <vm_ref_or_name> <7>
+    - id: <source_vm> <7>
 EOF
 ----
 <1> Specify the target cluster.
 <2> Specify the target namespace.
-<3> Specify the managed object reference of one or more VMware networks.
+<3> Specify the managed object reference or name of the source network.
 <4> Specify the target namespace.
-<5> Specify one or more VMware data stores to map to the target data store.
-<6> Specify the managed object reference of the data store.
-<7> Specify the managed object reference or the names of one or more VMware VMs.
+<5> You can map one or more VMware data stores to the target data store.
```

Comment:
I did not test that but you can pick different target storage classes for the different source datastores.
It should work.